client/src/components/App/Content/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -34,7 +34,7 @@ const COUNTY_LABEL = 'נפה'
 const SOURCE_ORGANIZATION_LABEL = 'מסגרת'
 const LANGUAGE_LABEL = 'שפה'
 
-const SignUpForm: React.FC<Props> = ({ handleSaveUser }: Props) => {
+const SignUpForm: React.FC<Props> = ({ defaultValues, handleSaveUser }: Props) => {
```

Comment:
I don't think `defaultValues` should be the parameter to determine whether the form fields are disabled or not. You can add a `disableFields` prop instead

Reply:
I disabled all the fields in case of 'reading mode' - I know I have to disable the fields when I get defaultValues so I use it as a condition instead of using another prop